### DISCUSSION TITLE
Mousewheel backwards when connecting to Windows 11 machine

### DISCUSSION THREAD

**Discussion by Some1Else83129** [rustdesk/rustdesk#5369] [Q&A]
Good day. 

I do want to say thank you for everything that you do.

Certain (But not all) machines I connect to (A Windows 11 desktop in particular) - the mouse wheel is backwards (Scrolling down moves the window up, scrolling up moves the window down).  This wasn't always the case (I'm not sure which version upgrade did it though).
Is there any settings I can look into to fix this?

Thanks!

**Comment by Some1Else83129** [comment]
Has this been missed? Is there a setting you can help me with?

**Comment by rustdesk** [comment]
@fufesou add a setting for this. Under current keyboard icon on menubar, do not forget global display settings for this.
